src/Helpers/helpers.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+if (! function_exists('escape_cdata')) {
```

Comment:
I'd rather not introduce a global function here. Please move this to a static function in a class called `Format`. I don't mind the long call (with a namespace) in a the Blade view.

Reply:
Fixed in 7c6c887